samples/maps/plotoptions/mappoint-label/demo.js

```diff
@@ -0,0 +1,56 @@
+Highcharts.mapChart('container', {
+    chart: {
+        map: 'custom/europe'
+    },
+    title: {
+        text: 'Labels marking POI\'s'
+    },
+    mapNavigation: {
+        enabled: true
+    },
+    legend: {
+        enabled: false
+    },
+    plotOptions: {
+        mappoint: {
+            dataLabels: {
+                align: 'left',
+                x: -12,
+                y: -5,
+                enabled: true,
+                shape: 'callout',
+                backgroundColor: 'rgb(0 0 0 / 0.7)',
+                borderRadius: 5,
+                style: {
+                    color: 'white'
+                }
+            },
+            marker: {
+                fillColor: '#ffffff',
+                lineColor: '#000000',
+                lineWidth: 1
+            },
+            tooltip: {
+                pointFormat: '{point.name} [{point.lon:.2f}, {point.lat:.2f}]'
+            }
+        }
+    },
+    series: [{
+        name: 'Basemap',
+        borderColor: '#A0A0A0',
+        nullColor: 'rgba(0 96 0 / 0.2)'
```

Comment:
I know we're no longer supporting IE, but if we changed the `'rgb(0 0 0 / 0.7)'` string format to `'rgba(0, 0, 0, 0.7)'`, we could still fully support IE.

Also (still nothing important) there are some inconsistencies like:
- [ ]  using `rgb` in one place and `rgba` in another
- [ ]  using hex colors in small letters and then big letters
- [ ]  using hex colors and then named color `'white'`